xrootd/resources/xrootd-origin.cfg

```diff
@@ -85,6 +85,6 @@ ofs.ckslib * libXrdMultiuser.so
 xrootd.chksum max 2 md5 adler32 crc32
 xrootd.trace emsg login stall redirect
 pfc.trace info
-pss.setopt DebugLevel 1
+pss.setopt DebugLevel 3
```

Comment:
Let's rebase this change away.  I think this debug level should be controlled by the configuration, not hardcoded.